igcollect/logfile_values.py

```diff
@@ -133,18 +133,17 @@ def get_distribution(self):
         return d
 
     def get_metric_value(self):
-
         if not self.values:
             return 0
 
         if not self.function:
             return float(self.get_last_value())
 
-        if 'distribution' in self.function:
+        if self.function == 'distribution':
             return getattr(self, 'get_' + self.function)()
 
-        if 'count_' in self.function:
-            if 'percentage' in self.function:
+        if self.function.startswith('count'):
```

Comment:
Why just `count` here all of a sudden and not `count_`? That will also now catch functions like `catchy_phrase`.

Reply:
And? What is a reason to put "catchy" phrase

Reply:
Sry, typo, actually it should have been `countable_phrase`. It just does not make sense to check if it is a `count_*` function by just checking `count` as prefix. Just add the `_` there again.